modules/kmm-setting-soft-dependencies-between-kernel-modules.adoc

```diff
@@ -11,7 +11,8 @@ These are called soft dependencies.
 `depmod` is usually not aware of these dependencies, and they do not appear in the files it produces.
 For example, if `mod_a` has a soft dependency on `mod_b`, `modprobe mod_a` will not load `mod_b`.
 
-You can resolve these situations by declaring soft dependencies in the Module Custom Resource Definition (CRD) using the `modulesLoadingOrder` field.
+You can resolve these situations by declaring soft dependencies in the Module Custom Resource
+Definition (CRD) using the `modulesLoadingOrder` field.
```

Comment:
I also think the term can be spelled with lowercase letters:
```suggestion
You can resolve these situations by declaring soft dependencies in the Module custom resource
definition (CRD) using the `modulesLoadingOrder` field.
```

Reply:
Changed.